salty-747/html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/FMC/B747_8_FMC_ProgPage.js

```diff
@@ -145,22 +148,59 @@ class B747_8_FMC_ProgPage {
         else {
             crzSpeedCell = Simplane.getAutoPilotAirspeedHoldValue().toFixed(0);
         }
+        let todTimeCell = "";
+        let todDistanceCell = "";
+        let toToDText = "";
+        let distanceToTOD = SimVar.GetSimVarValue("L:WT_CJ4_TOD_REMAINING", "number");
+        if (distanceToTOD > 0) {
+            if (isFinite(distanceToTOD) && distanceToTOD < 201) {
+                toToDText = "TO T/D";
+                if (distanceToTOD < 5) {
+                    todDistanceCell = "NOW";
+                } else {
+                    todDistanceCell = distanceToTOD.toFixed(0)  + "NM";
+                    todTimeCell = "&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp";
+                    let eta = undefined;
+                    eta = (B747_8_FMC_ProgPage.computeEtaToWaypoint(distanceToTOD, speed) + currentTime) % 86400;
```

Comment:
no need to assign it to undefined first, just assign it straight away. also, const instead of let